api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -23,6 +23,10 @@ message FilterTransformations {
   // Only RouteTransformations.RouteTransformation with matching stage will be
   // used with this filter.
   uint32 stage = 2 [ (validate.rules).uint32 = {lte : 10} ];
+
+  // If set to true, the filter will log the request/response body and headers before 
+  // any transformation is applied.
+  bool log_request_response_info = 3;
```

Comment:
this will not sanitize anything, yes? feels very similar in spirit to our closed source enterprise graphql logging API: https://github.com/solo-io/envoy-gloo-ee/blob/main/api/envoy/config/filter/http/graphql/v2/graphql.proto#L438-L440 we should be very explicit about the potential security risks of turning this on (logging aggregators persisting potentially sensitive information)

Reply:
Yeah, this PR is pretty directly based off of those changes. I clarified the potential risks in this commit: https://github.com/solo-io/envoy-gloo/pull/240/commits/1bfb840c09c77fe810936e557e53f1cb4c54db44

```diff
+ // Logs request/response sensitive information
+ // By default, this is false so no request or response sensitive information is logged.
  // If set to true, the filter will log the request/response body and headers before 
  // any transformation is applied.
  bool log_request_response_info = 3;
```